### ISSUE TITLE
Support for "include Rules"

### ISSUE THREAD

**Issue by igordejanovic** [igordejanovic/textX#13]
A form of rules that will generate attributes on the calling meta-class.

```
Rule1:
    a=INT Rule2;
Rule2:
    b=STRING;
```
`Rule2` is called directly (not from an assignment) thus `b` attribute will be created on `Rule1` instance. `Rule2` instance will not be instantiated.

**Comment by danibishop** [comment]
I was just about to ask about this 👍 

In my case, I just expected this as default behavior and I guess I would not be the only one.

**Comment by igordejanovic** [comment]
Yep. Definitely should be the default behaviour. It would reduce grammar boilerplate in some cases.
